### ISSUE TITLE
Make styleguide URL configurable

### ISSUE THREAD

**Issue by Wolfr** [usebedrock/bedrock#410]
I implemented a way to make the styleguide URL configurable in https://github.com/smartcoop/design/commit/1309442d41aa076656d1c1f9b447bb9826041f81 .

Need to port this to the main Bedrock logic, because it is useful if you want to make a website with a different layout than “standard” Bedrock.

**Comment by Wolfr** [comment]
I implemented this in https://github.com/usebedrock/bedrock/tree/feature/styleguide-configurable-url .

I had to remove a long standing “hack” where we would redirect `/styleguide` to `/styleguide/docs/introduction.html`.

Now I had the problem that the content of that page could not be changed; so I added in a mixin logic for now that calls a global mixin. This feels hacky but I am not sure how to resolve it in a cleaner way.

(Content has to live in `content` whereas the “app” itself has to live in core)

As a result of not being a part of the docs templates, the “introduction” is not listed in the tree anymore. This might be a positive outcome for people who want to fully customize how their design system site looks like, like we did at https://design.smart.coop/ .
